app/controllers/course/forum/posts_controller.rb

```diff
@@ -9,8 +9,7 @@ class Course::Forum::PostsController < Course::Forum::ComponentController
 
   def create
     if super
-      # Update parent topic updated_at to invalidate all read marks
-      @topic.touch
+      @topic.update_column(:latest_post_at, @post.created_at)
```

Comment:
Actually can we wrap this up in the same transaction as the creation of the post? I'm not sure whether we can have this as a callback within the posts controller as well ... any thoughts?

Reply:
Not sure if we can do that, this is for `Course::Forum::Topic` while the posts concern deals with the abstract `Course::Discussion::Topic` class.

Reply:
This is only for forum topic, I think we have to put it here. (we dont have a forum post model)